script/entrypoint.sh

```diff
@@ -20,9 +19,13 @@ fi
 export \
   AIRFLOW_HOME \
   AIRFLOW__CORE__EXECUTOR \
-  AIRFLOW__CORE__FERNET_KEY \
   AIRFLOW__CORE__LOAD_EXAMPLES \
 
+# Setup the Fernet Key only if the user didn't provide it explicitely as an Airflow configuration
+if [[ -z "$AIRFLOW__CORE__FERNET_KEY" && -z "$AIRFLOW__CORE__FERNET_KEY_CMD" ]]; then
```

Comment:
This can cause the user to lose data if they forget to keep the Fernet key. For Airflow 1.10, the fernet key is optional. For Airflow 2.0, it is required.